Kernel version parsing dies on RHEL 7

From a run using "marketplace_image:RedHat RHEL 7-RAW latest" 

```
lisa.env[generated_0].node[0].cmd[5907] not found command: Command not found: waagent. 
Check that waagent is installed and on $PATH
2021-08-25 23:02:03.525[26188][ERROR] lisa.env[generated_0] failed to get environment information

Traceback (most recent call last):

  File "G:\lisadpdk\lisa\environment.py", line 458, in get_environment_information
    linux_information = uname.get_linux_information()
  File "G:\lisadpdk\lisa\base_tools\uname.py", line 59, in get_linux_information
    kernel_version=parse_version(match_result.group("kernel_version")),
  File "G:\lisadpdk\lisa\util\__init__.py", line 315, in parse_version
    ver: Dict[str, Any] = {
  File "G:\lisadpdk\lisa\util\__init__.py", line 316, in <dictcomp>
    key: 0 if value is None else int(value)
ValueError: invalid literal for int() with base 10: '1062.1.1.el7.x86_64'
```

--- a/lisa/sut_orchestrator/azure/platform_.py
+++ b/lisa/sut_orchestrator/azure/platform_.py
@@ -892,14 +892,13 @@ class AzurePlatform(Platform):
         arm_parameters.storage_name = get_storage_account_name(
             self.subscription_id, arm_parameters.location
         )
-        if arm_parameters.enable_sriov is None:
+        if node.capability.has_feature(features.Sriov.name()):
             arm_parameters.enable_sriov = True
-            if node.capability.has_feature(features.Sriov.name()):
-                self._log.debug(
-                    "use synthetic network since used size doesn't have "
-                    "sriov capability"
-                )
-                arm_parameters.enable_sriov = False
+        else:
+            self._log.debug(
+                "use synthetic network since used size doesn't have Sriov capability"
+            )
+            arm_parameters.enable_sriov = False
 
         # the arm template may be updated by the hooks, so make a copy to avoid
         # the original template is modified.

--- a/lisa/util/__init__.py
+++ b/lisa/util/__init__.py
@@ -315,7 +315,9 @@ def parse_version(version: str) -> VersionInfo:
     ver: Dict[str, Any] = {
         key: 0 if value is None else int(value)
         for key, value in match.groupdict().items()
+        if key != "prerelease"
     }
+    ver["prerelease"] = match["prerelease"]
     rest = match.string[match.end() :]  # noqa:E203
     ver["build"] = rest
     release_version = VersionInfo(**ver)

